doc/chapters/api/index.rst

```diff
@@ -7,105 +6,291 @@ API
 This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
 mostly useful for integrators.
 
-.. TODO::
-    This chapter is still to be written properly. Some bits and pieces have
-    been but they are under development.
-
 .. _dbus-api:
 
-DBus API
+D-Bus API
 ========
 
+D-Bus API is an IPC interface to call SoftwareContainer agent methods. The API provides following path and interface.
+
 :Path: /com/pelagicore/SoftwareContainer
 :Interface: com.pelagicore.SoftwareContainerAgent
 
+In this document path and interface above used in all method call examples below with following order.::
+
+        dbus-send --system --print-reply \
+        --dest=Interface \
+        Path \
+        Method \
+        Arguments
+
 Methods
 -------
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        *None*
+|
+:Return Value:
+        *None*
+|
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
 
 CreateContainer
 ---------------
-:containerIDPrefix: string
-:config: string
-
-uint32 containerID CreateContainer(string containerIDPrefix, string config)
+Creates a new container and returns created container id.
+
+:Parameters:
+        :IDPrefix: ``string`` prefix for random container id.
```

Comment:
Currently, this prefix is part of the container name (in the LXC domain), and I think referring to it as "ID" in these parts of the docs might confuse the reader to think it has something to do with the container ID used to identify containers when using this API.